### PR TITLE
Issue 86

### PR DESCRIPTION
**Pull request by ibaldin**
Fixes tracing of parent nodes in ad topologies. Also fixes printing capacities in ads if nothing is allocated.

### CODE REVIEW

**Comment by ibaldin** [comment]
Addresses #86

File: fim/user/topology.py [hunk 1]
```diff
@@ -750,6 +754,14 @@ def __list_links(self) -> ViewOnlyDict:
     def sites(self):
         return self.__list_sites()
 
+    @property
+    def nodes(self):
+        """
+        Same as sites property
+        :return:
+        """
+        return self.__list_sites()
```
Comment (kthare10) [inline fim/user/topology.py:763]: Question: Is this intended to return sites?

**Review by kthare10 — APPROVED** [review]
Looks good, I just left a question.

File: fim/user/topology.py (hunk 1 above)
Comment (ibaldin) [reply fim/user/topology.py:763]: Yes in advertised topologies it is the same as .nodes property.

**Pull request merged**